### PR TITLE
WIP: Yg/add parking vehicle detector

### PR DESCRIPTION
**Pull request by ygnn123**

### CODE REVIEW

File: models/public/parking-vehicle-detection/model.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2019 Intel Corporation
```
Comment (eizamaliev) [inline models/public/parking-vehicle-detection/model.yml:1]:
```suggestion
# Copyright (c) 2020 Intel Corporation
```

**Comment by eizamaliev** [comment]
@ygnn123 Can your model be converted by the `pytorch_to_onnx.py` script?

File: models/public/parking-vehicle-detection/model.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    size: tbd
+    sha256: tbd
+    source: tbd
+model_optimizer_args:
+  - --input_shape=[1,3,800,1333]
+  - --input=image
+  - --mean_values=data[123.675, 116.28, 103.53]
+  - --scale_values=data[58.395, 57.12, 57.375]
+  - --input_model=$dl_dir/mobilenet-ssd.caffemodel
+  - --input_proto=$dl_dir/mobilenet-ssd.prototxt
```
Comment (eaidova) [inline models/public/parking-vehicle-detection/model.yml:38]: @ygnn123 do not forget to update these lines, according your template above:
```suggestion
  - --input_model=$dl_dir/parking-vehicle-detection.onnx
```

Comment (eizamaliev) [reply models/public/parking-vehicle-detection/model.yml:38]: If there are no problems with PyTorch to ONNX conversion, it's better to download PT model.

File: models/public/parking-vehicle-detection/parking-vehicle-detection.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  The "Cascade-RCNN" model is two-stage multibox Detection network intended to perform 
+  object detection. This model is implemented using mmdetection framework.
+
+  The model input is a blob that consists of a single image of 1x3x800x1333 in BGR
+  order. The BGR mean values need to be subtracted
+  as follows: [123.675, 116.28, 103.53] before passing the image blob into the network. 
+  Also BGR values need to be scaled by division on following coefficients: [58.395, 57.12, 57.375]  
+
+
+## Example
```
Comment (vladimir-dudnik) [inline models/public/parking-vehicle-detection/parking-vehicle-detection.md:13]: we are removing empty sections from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please provide sample image or remove empty section

File: models/public/parking-vehicle-detection/parking-vehicle-detection.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+| Type              | Detection     |
+| GFLOPs            | 234.46        |
+| MParams           |   tbd         |
+| Source framework  | Pytorch\*     |
+
+## Accuracy
+
+TBD
+
+## Performance
```
Comment (vladimir-dudnik) [inline models/public/parking-vehicle-detection/parking-vehicle-detection.md:28]: please remove empty section

File: models/public/parking-vehicle-detection/parking-vehicle-detection.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 234.46        |
+| MParams           |   tbd         |
+| Source framework  | Pytorch\*     |
+
+## Accuracy
+
+TBD
```
Comment (vladimir-dudnik) [inline models/public/parking-vehicle-detection/parking-vehicle-detection.md:26]: please provide expected accuracy metric value

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

**Pull request closed without merging**